website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -246,6 +250,11 @@ To update:
 
 </Tabs>
 
+## dbt Cloud CLI extensions
+Command line extensions are additional functionalities that you can add to a command line tool or shell environment. These extensions expand the capabilities of the CLI, making it more versatile and efficient.
+
+The dbt Cloud CLI doesn't currently support extensions, such as SQLFluff or dbt-power-user, during the public preview period. This is because.....
```

Comment:
I would rewrite this section as:

## Using VS Code extensions with the Cloud CLI
The Cloud CLI is fully compatible with dbt Core, but does not provide certain Core APIs that some tools, particularly VS Code extensions, need to run. 

If you need to use VS Code extensions like dbt-power-user with the Cloud CLI, install via the brew method alongside a dbt Core installation and alias the Cloud CLI to run as `dbt-cloud`. This will allow dbt-power-user to continue to function alongside the Cloud CLI.